jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt

```diff
@@ -328,73 +317,37 @@ class GettingStartedPanel(
-                                            FeatureId.Codecatalyst,
-                                            CredentialSourceId.AwsId,
-                                            isAggregated = true,
-                                            Result.Succeeded
-                                        )
-                                        AuthTelemetry.addedConnections(
-                                            project,
-                                            getSourceOfEntry(SourceOfEntry.CODECATALYST, isFirstInstance, connectionInitiatedFromExplorer),
-                                            oldConnectionCount,
-                                            getConnectionCount() - oldConnectionCount,
-                                            enabledAuthConnections = initialEnabledConnection,
-                                            newEnabledAuthConnections = getEnabledConnections(project).toString(),
-                                            attempts = 1,
-                                            Result.Succeeded
-                                        )
-                                    } else {
-                                        controlPanelVisibility(panelConnectionInProgress, panelNotConnected)
-                                        AuthTelemetry.addConnection(
-                                            project,
-                                            getSourceOfEntry(SourceOfEntry.CODECATALYST, isFirstInstance, connectionInitiatedFromExplorer),
-                                            FeatureId.Codecatalyst,
-                                            CredentialSourceId.AwsId,
-                                            isAggregated = false,
-                                            Result.Failed,
-                                            reason = "Browserloginfailure"
-                                        )
-                                        AuthTelemetry.addedConnections(
-                                            project,
-                                            getSourceOfEntry(SourceOfEntry.CODECATALYST, isFirstInstance, connectionInitiatedFromExplorer),
-                                            oldConnectionCount,
-                                            getConnectionCount() - oldConnectionCount,
-                                            enabledAuthConnections = initialEnabledConnection,
-                                            newEnabledAuthConnections = getEnabledConnections(project).toString(),
-                                            attempts = 1,
-                                            Result.Failed
-                                        )
-                                    }
+                                    handleCodeCatalystLogin(requestCredentialsForCodeCatalyst(project), panelNotConnected)
```

Comment:
can we also add the 2 lines of telemetry in this function?

Reply:
Telemetry is handled in `requestCredentialsForCodeCatalyst`. I removed them from here since the CW panel also behaves this way.